modules/applications-create-using-cli-modify.adoc

```diff
@@ -199,3 +208,18 @@ To search for images, templates, and other inputs for the `oc new-app` command,
 ----
 $ oc new-app --search php
 ----
+
+[id="setting-the-import-mode"]
+== Setting the import mode 
+
+To set the import mode when using `oc new-app`, add the `--import-mode` flag. This flag can be appended with `Legacy` or `PreserveOriginal`, which provides users the option to create image streams using a single sub-manifest, or all manifests, respectively. 
```

Comment:
I would maybe add either `ImportMode` values of before or values after `legacy` or PreserveOriginal`.